app/articles/server/methods/admin.js

```diff
@@ -0,0 +1,87 @@
+import { Meteor } from 'meteor/meteor';
+import { HTTP } from 'meteor/http';
+
+import { API } from '../utils/url';
+import { settings } from '../../../settings';
+
+const api = new API();
+
+function setupGhost(user, token, rcUrl) {
+	const blogTitle = settings.get('Article_Site_title');
+	const data = {
+		setup: [{
+			rc_url: rcUrl,
+			rc_id: user._id,
+			rc_token: token,
+			name: user.name,
+			email: user.emails[0].address,
+			password: 'qwe123qwe123', // TODO send random password; remove password field
+			blogTitle,
+		}],
+	};
+	return HTTP.call('POST', api.setup(), { data, headers: { 'Content-Type': 'application/json' } });
+}
+
+function createSession(options) {
+	const response = HTTP.call('POST', api.session(), options);
+	if (response.statusCode === 201 && response.content === 'Created' && response.headers && response.headers['set-cookie']) {
+		let cookie = response.headers['set-cookie'][0].split(';');
+		cookie = `${ cookie[0] };${ cookie[1] };${ cookie[2] };${ cookie[4] }`;
```

Comment:
Can you please explain what's happening here?

Reply:
Basically, it is setting the cookies for the domain, with HttpOnly option false.
Without it, I was unable to fetch cookies on cross-port